CHANGELOG.md

```diff
@@ -6,7 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [9.0.0]
+## [10.0.0]
+### Uncategorized
+- **BREAKING:** Update module name to use `@metamask` scope ([#187](https://github.com/MetaMask/KeyringController/pull/187))
+  - Consumers will now need to import this package as `@metamask/eth-keyring-controller` 
+- Bump @metamask/eth-hd-keyring to v6.0.0 ([#193](https://github.com/MetaMask/KeyringController/pull/193))
```

Comment:
not clear to me whether this should be considered breaking or not... technically we don't expose the serialized keyrings to consumers it could be considered an implementation detail?

Reply:
Hmm, maybe best to describe it as breaking so that it's clear users of the library probably need to make some change to accommodate it, or consider the change carefully in some way. It will have a functional impact on anyone relying on serialization.

```suggestion
- **BREAKING:**  @metamask/eth-hd-keyring to v6.0.0 ([#193](https://github.com/MetaMask/KeyringController/pull/193))
  - The serialization format has changed back to the format used in v4.x.
```

Reply:
>technically we don't expose the serialized keyrings to consumers it could be considered an implementation detail?

It's included in the KeyringController `store`, which includes the encrypted vault. So it is exposed directly in that sense.